.scalafix.conf

```diff
@@ -0,0 +1,10 @@
+rules = [
+  # DisableSyntax
+  # ExplicitResultTypes
+  #LeakingImplicitClassVal
+  RemoveUnused
+  #NoAutoTupling 
+  #NoValInForComprehension 
+  #ProcedureSyntax
```

Comment:
```suggestion
  RemoveUnused
```

Reply:
I should just delete this file, it's unused in this PR